### DISCUSSION TITLE
Add batch downloading and automatic detection and addition of and paste url addresses of the clipboard

### DISCUSSION THREAD

**Discussion by trimechee** [setvisible/DownZemAll#202] [Ideas]
Hello, sometimes we need to download quickly specific videos from a YouTube channel, so to speed up and simplify the process and avoid clicking on "paste link" each time, it would be great please if our beloved ArrowDL could detect the url address contained in the clipboard and automatically add it and paste it to the batch downloader, thank you so much !

**Comment by setvisible** [comment]
Hi

This feature is already implemented:

When you open the dialog "batch" or "stream", the url that is in the clipboard is automatically fetched and pasted to the dialog.

Regards
